scripts/cargo-msfs.ts

```diff
@@ -85,36 +67,33 @@ if (installedSdks.versions.some(v => !v.up_to_date)) {
 
 // Clear out dir
 const outDir = resolve(workspaceRoot, "dist/wasm");
+
 if (existsSync(outDir)) rmdirSync(outDir, { recursive: true });
 
 // The work directory, relative to workspace root
 const relativeWorkdDir = process.cwd().replace(workspaceRoot, "").replaceAll("\\", "/");
 
-// Build the selected versions
-await Promise.all(
-  versionsToBuild.map(async simVersion => {
-    console.info(`[*] Building for ${simVersion}`);
-
-    // Create the subfolder
-    const simDir = join(outDir, simVersion);
-    const relativeSimDir = simDir.replace(workspaceRoot, "").replaceAll("\\", "/");
-    mkdirSync(simDir, { recursive: true });
-
-    const color = simVersion === "2020" ? "\x1b[34m" : "\x1b[32m";
-
-    // Run cargo-msfs
-    await $`docker run \
-      --rm -t \
-      --name msfs-${simVersion}-wasm-builder \
-      -v ${workspaceRoot}:/workspace \
-      -w /workspace${relativeWorkdDir} \
-      -e CARGO_TARGET_DIR=/workspace/targets/${simVersion} \
-      ${IMAGE_NAME} \
-        bash -c "cargo-msfs build msfs${simVersion} -i ./src/wasm -o ./${relativeSimDir}/msfs_navigation_data_interface.wasm \
-    1> >(sed \"s/^/[${color}${simVersion}\\x1b[0m]/\") \
-    2> >(sed \"s/^/[${color}${simVersion}\\x1b[0m]/\" >&2)"`.catch((err: { exitCode?: number; stderr?: Buffer }) => {
-      console.error(`[-] Error building for ${simVersion}: ${err.exitCode} ${err.stderr?.toString()}`);
-      process.exit(1);
-    });
-  }),
-);
+// Build the 2020 version
+const simVersion = "2020";
```

Comment:
Minor nitpick: Is having this as a variable still necessary? i.e. could we just inline `2020` in the places we use this?